caffe2/proto/caffe2.proto

```diff
@@ -184,6 +243,34 @@ message OperatorDef {
   // This is an optional string with no assumed characteristics as
   // operators can be constructed in any language.
   optional string debug_info = 10;
+
+  // the domain of the operator
+  // to support legacy operator libs, we use domain to distinguish which
+  // operator lib we refer to:
+  //   - "CAFFE2" means this uses Caffe2 operator library
+  //   - "ATEN" means this uses ATen operator library
+  //   - "C10" is for the fused library
+  //   - if the domain is missing or empty, we use "CAFFE2", this is for
+  //     legacy models
+  optional string domain = 11;
+  // each operator is has its own version number.
+  // operator version information
+  // each time, we change the API or semantics of the operator,
+  // we bump the version for the operator.
+  // the runtime system should check the op_version of each OperatorDef
+  // and decide it should reject or accept the model
+  optional int64 op_version = 12;
+
+  // annotation is a list of named values for additional information
```

Comment:
since it's commented out - just remove it?

Reply:
yes, will remove it.